tests/apps/populate_reentry/user_script.py

```diff
@@ -0,0 +1,7 @@
+from django import setup
+
+
```

Comment:
remove blank line per isort (there's an isort bug that prevents the check from flagging this, but if you run isort manually it'll fix it)